src/main/java/nl/tudelft/contextproject/model/Bomb.java

```diff
@@ -69,9 +70,17 @@ public void setSpatial(Spatial spatial) {
 
 	@Override
 	public PhysicsControl getPhysicsObject() {
+		if (rb != null) return rb;
 		CollisionShape sceneShape = CollisionShapeFactory.createMeshShape(sp);
-		RigidBodyControl rigidBody = new RigidBodyControl(sceneShape, 0);
-		rigidBody.setPhysicsLocation(sp.getLocalTranslation());
-		return rigidBody;
+		rb = new RigidBodyControl(sceneShape, 0);
+		rb.setPhysicsLocation(sp.getLocalTranslation());
+		return rb;
+	}
+
+	@Override
+	public void move(float x, float y, float z) {
```

Comment:
This seems to be code which shouldn't be merged with this feature. Can anyone confirm?